Dokan/Dokan/Scenes/ReviewProduct/ReviewProductNavigationBarBehavior.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ReviewProductNavigationBarBehavior.swift
+//  Dokan
+//
+//  Created by marwa on 18/08/2022.
+//
+
+import Foundation
+import UIKit
+
+class ReviewProductNavigationBarBehavior {
+
+    // MARK: - Properties
+
+    private unowned var navigationItem: UINavigationItem
+    private var onBack: () -> Void = {}
+
+    // MARK: - Init
+
+    init(navigationItem: UINavigationItem) {
+        self.navigationItem = navigationItem
+    }
+
+    // MARK: - Configuration
+
+    func configure(_ rating: Double, onBack: @escaping () -> Void) {
+
+        // MARK: Add back button to navigation bar
+
+        self.onBack = onBack
+        let backButton = UIBarButtonItem(image: UIImage(named: "left")?.withRenderingMode(.alwaysOriginal),
+                                         style: .done,
+                                         target: self,
+                                         action: #selector(backWasTapped))
+
+        navigationItem.leftBarButtonItem = backButton
+
+        // MARK: Add star image and rating label
+
+        lazy var ratingStackView: UIStackView = {
```

Comment:
No need for lazy since it should be used in the next line. 
```suggestion
        let ratingStackView: UIStackView = {
```